lib/unit_tests_utils/postgresql_client.rb

```diff
@@ -0,0 +1,54 @@
+require 'pg'
+
+class PostgreSQLClient
+  attr_reader :host, :dbname, :user, :password, :sslmode
+
+  def initialize(args)
+    @host = args[:host]
+    @hostaddr = args[:hostaddr]
+    @dbname = args[:dbname] || 'area51'
+    @user = args[:user]
+    @password = args[:password]
+    @sslmode = args[:sslmode] || 'disable'
+  end
+
+  def self.create_from_manifest(manifest, host = nil)
+    host = manifest.hostname unless host
+    user = manifest.properties['postgresql-ha']['admin_credentials']['username']
+    password = manifest.properties['postgresql-ha']['admin_credentials']['password']
+    if manifest.properties['postgresql-ha']['ssl']['enable']
```

Comment:
It is ok for this PR, because the manifest is specifying the `ssl.enable` property.

But, we should avoid accessing the properties like a nested map, since if the ssl property is not provided, this would break the test.

To make our life easier, there is a traversal search that can return nil if the full path does not exists using the ops file notation: https://github.com/anynines/postgresql-boshrelease/blob/master/unit-tests/spec/support/test_helper.rb#L13

Reply:
Okay done!